pandas/io/formats/style.py

```diff
@@ -1222,6 +1222,76 @@ class MyStyler(cls):
 
         return MyStyler
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs), and return the result.
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        func : function
+            function to apply to the Styler.
+            ``args``, and ``kwargs`` are passed into ``func``.
+            Alternatively a ``(callable, data_keyword)`` tuple where
+            ``data_keyword`` is a string indicating the keyword of
+            ``callable`` that expects the Styler.
+        args : iterable, optional
+            positional arguments passed into ``func``.
+        kwargs : mapping, optional
```

Comment:
Add the `*` before args and kwargs. I'd prefer `dict` intead of mapping. Parameter descriptions should start with a capital letter.

If you run `./scripts/validate_docstrings.py pandas.Styler.pipe` you should get a report with most of the docstring issues.

Reply:
Done.  As mentioned above, now validating.